src/tools/spark.md

```diff
@@ -70,22 +70,43 @@ Example of using the Storage API from Databricks:
 dbutils.library.installPyPI("google-cloud-bigquery", "1.16.0")
 dbutils.library.restartPython()
 
-# Read one day of pings and select a subset of columns.
+from google.cloud import bigquery
+
+
+def get_table(view):
+    bq = bigquery.Client()
```

Comment:
```suggestion
    """Helper for determining what table underlies a user-facing view, since the Storage API can't read views."""
    bq = bigquery.Client()
```